packages/ui/src/Popup/AuthManagement/WebsiteEntry.tsx

```diff
@@ -19,15 +21,21 @@ function WebsiteEntry ({ className = '', info, toggleAuth, url }: Props): React.
 
   return (
     <div className={`${className} ${info.isAllowed ? 'allowed' : 'denied'}`}>
-      <div className='url'>
-        {url}
-      </div>
+      <Box pr={10}>
+        <Icon Asset={SvgWeb}
+          color='primary'
+          height={20}
+          width={20}
+        />
+      </Box>
+      <Text style={{ flex: 1 }}
```

Comment:
Might have to adjust the sizes a bit, or play with the flex properties, there's a bit of an overflow I noticed

<img width="400" src="https://user-images.githubusercontent.com/7417976/116475154-7096af80-a847-11eb-8bcd-ecc12d164e95.png" />

Reply:
Just noticed the `<span style={{ width: 40 }}>` for the label on the switch, I believe that's probably what's leading to the overflow. Also, this may make the switch less re-usable in other places due to the locked width.

Without the specified width on the label, however, I notice that elements "move" around when the text changes. Adding a wrapper around the switch in WebsiteEntry with a specified width solves the overflow issue and keeps elements in place!

Reply:
Thanks. I removed this since it's not needed anymore.